cdi/pom.xml

```diff
@@ -167,12 +166,13 @@
                 <artifactId>maven-bundle-plugin</artifactId>
                 <configuration>
                     <instructions>
-                        <Fragment-Host>org.hibernate.validator</Fragment-Host>
+                        <Bundle-SymbolicName>${hibernate-validator-cdi.module-name}</Bundle-SymbolicName>
+                        <Fragment-Host>${hibernate-validator.module-name}</Fragment-Host>
                         <Import-Package>
-                            org.hibernate.validator.*;version="[${project.version},${project.version}]",
+                            javax.validation.*;version="[2.0,3.0)",
```

Comment:
I think we should inherit this from the fragment host. On the `bootstrap` package I've asked Arjan on the issue why it is that it's needed. It seems odd as we don't reference that class.